charon/src/translate/translate_functions_to_ullbc.rs

```diff
@@ -1462,16 +1463,21 @@ impl<'tcx, 'ctx, 'ctx1> BodyTransCtx<'tcx, 'ctx, 'ctx1> {
         mut self,
         rust_id: DefId,
         arg_count: usize,
+        item_meta: &ItemMeta,
     ) -> Result<Option<ExprBody>, Error> {
         // Stopgap measure because there are still many panics in charon and hax.
```

Comment:
Please put attention to the comments when you update the code. This comment is about the `panic::catch_unwind` and should have been moved with it: it has nothing to do with the check `if item_meta.opaque`.